ajax/libs/venn.js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "venn.js",
+  "filename": "venn.min.js",
+  "description": "A javascript library for laying out area proportional venn and euler diagrams",
+  "homepage": "https://github.com/benfred/venn.js/",
+  "author": "Ben Frederickson",
+  "keywords": [
+    "venn-diagram",
+    "javascript",
+    "d3",
+    "euler-diagram"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/benfred/venn.js.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/benfred/venn.js.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "venn*"
```

Comment:
Let's use git auto-update config because there are more versions from npm package then Github.
The whole auto-update config as the following:
```js
  "npmName": "venn.js",
  "npmFileMap": [
    {
      "basePath": "build",
      "files": [
        "**/!(bundle.*|*.zip)"
      ]
    },
    {
      "basePath": "",
      "files": [
        "venn*"
      ]
    }
  ]
```

Thanks.